### DISCUSSION TITLE
Github for distributing audio and video

### DISCUSSION THREAD

**Discussion by GezzMC** [community/community#21891] [New to GitHub]
<p>I did some searches but didn’t find anything about this. Authoritatively, only github staff could answer this question, and even then, because supporting audio and music isn’t GitHub’s core service, maybe not, because no one has thought about it.</p>
<p>There are lots of sites devoted to distributing music, but for various reasons I don’t like how they work. GitHub, with git, is obviously perfect for syncing local files to a distribution site. Because of that, I’d like to use it for a blog which is used to distribute audio and video files.</p>
<p>But, again, audio and video isn’t GitHub’s core service, and audio and video files are larger than text files. So, I assume that if I actually got a lot of traffic that was pulling the audio and video files, my repository would get shut down.</p>
<p>In the past, I read where someone was using GitHub to host the distribution of their music.</p>
<p>Can anyone tell me whether GitHub officially allows distributing audio and video files?</p>
<p>Thanks.</p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/gezzmc">@gezzmc</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/gezzmc">@gezzmc</a>,</p>
<p>Thanks for being part of the GitHub Community Forum! </p>
<p>I think we’d be better able to answer your question if you write to <a>private support</a>. That way, we can look at specifics of your request/question and view your specific account (which we can’t discuss in a public forum, for privacy and safety reasons). Please feel free to link to this forum post when writing in so that you don’t have to write out your question all over again.</p>
<p>Thanks again for being here!</p>
<p>Cheers!</p>

**Comment by GezzMC** [comment]
<p><a class="mention" href="/angela-crist">@angela-crist</a>, thanks for moving it to a better place.</p>
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a>, thanks. I thought I’d do general-board thing first before going solo.</p>
